### DISCUSSION TITLE
The actual shm-size usage does not match the calculation on my system

### DISCUSSION THREAD

**Discussion by hua-bing** [blakeblackshear/frigate#14746] [Ask A Question] [answered]
I am playing with 0.15.0-beta1 using one 1080P camera (Wyze Cam V2 on wz_mini_hacks).  My system is a QNAP NAS (J3455 with 16G RAM).  The container was created following [QNAP install document](https://deploy-preview-13787--frigate-docs.netlify.app/frigate/installation).  According to the document, I should allocate at least 60 M shared memory:
```
python -c 'print("{:.2f}MB".format(((1280 * 1080 * 1.5 * 10 + 270480) / 1048576) + 40))'
60.03MB
```
However, when I spined off the docker container and checked shm usage under 'System/Storage', I noticed that it actually used more than twice as much (see attached picture).
![shm_usage_by_1_camera](https://github.com/user-attachments/assets/f01d08f8-ab2c-4f7d-a3e6-564188e2b28e)

I also tried with 6 same kind of cameras (expect 160.20MB shm according to the formula) and found that my 940 M shm got almost filled up (screenshot not provided because I think above screenshot is sufficient to show the issue).

I haven not observed any issue though after a couple of hours.  Just curious to know what is going on.

**Comment by NickM-27** [comment, marked as answer]
we will need to update the docs, that calculation is no longer accurate

**Reply by hua-bing** [reply]
Thanks for checking.
